main/gcc/APKBUILD

```diff
@@ -245,7 +245,7 @@ build() {
 	armv7-*-*-*eabihf)	_arch_configure="--with-arch=armv7-a --with-tune=generic-armv7-a --with-fpu=vfpv3-d16 --with-float=hard --with-abi=aapcs-linux --with-mode=thumb";;
 	mipsel-*-*-*)		_arch_configure="--with-arch-32=mips2 --with-tune-32=mips32 --with-fp-32=32 --with-mips-plt --with-float=hard --with-abi=32";;
 	powerpc-*-*-*)		_arch_configure="--enable-secureplt --enable-decimal-float=no";;
-	powerpc64-*-*-*)	_arch_configure="--with-abi=elfv2 --enable-secureplt --enable-decimal-float=no";;
+	powerpc64le-*-*-*)	_arch_configure="--with-abi=elfv2 --enable-secureplt --enable-decimal-float=no";;
```

Comment:
I have changed this part because the previous way had no effect whatsoever when building in a ppc64le machine.
The way it is now works.